x-pack/plugins/security_solution/public/management/pages/blocklist/view/components/blocklist_form.tsx

```diff
@@ -165,6 +168,20 @@ export const BlockListForm = memo(
       return selectableFields;
     }, [selectedOs]);
 
+    const valueLabel = useMemo(() => {
+      return (
+        <div>
+          <EuiFormLabel>{VALUE_LABEL} </EuiFormLabel>
+          <EuiIconTip
+            content={VALUE_LABEL_HELPER}
+            iconProps={{
+              className: 'eui-alignTop',
+            }}
+          />
```

Comment:
I'd prefer if the icon was on the top right of the input. Not sure if everyone else agrees. 
Should use `labelAppend` prop of `EuiFormRow`.   

Something like,
```
<EuiFormRow
        label={VALUE_LABEL}
        labelAppend={
         <EuiIconTip
            content={VALUE_LABEL_HELPER}
            iconProps={{
              className: 'eui-alignTop',
            }}
          />
        }
      >
```

Reply:
so I went with label here instead because the input is full width. when it's on a large screen, it looks a bit weird to me to leave it on the far right. thoughts @caitlinbetz?

Reply:
off the top of my head, I think I often see icons next to the label (as it currently is in the screen shot). So I'm okay with how it is currently unless we have a specific reason to move it!